### PR TITLE
HTTP Status Code instant app; fixes #283

### LINKED ISSUE #283: [Proposal]  HTTP status Codes
# Summary
 Returns a description of http status codes 

# Example search queries

- 401 status
- 501 code

### PR DESCRIPTION
**Pull request by Reckon77**
Added status codes from 100 - 308. I will make a commit for the rest of the status codes soon.

### CODE REVIEW

**Comment by Reckon77** [comment]
Fixes #283 

Hi please review and tell me if there are any changes required

File: apps/http-status-codes/src/files/codes.js [hunk 1]
```diff
@@ -0,0 +1,377 @@
+//data imported from MDN - https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
+const status_codes=[
+    {
+        value:"100",
+        meaning:"Continue",
+        description:"This interim response indicates that the client should continue the request or ignore the response if the request is already finished."
+    },
```
Comment (sahil-shubham) [inline apps/http-status-codes/src/files/codes.js:7]: Did you write all these yourself?

File: apps/http-status-codes/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  `
+const Meaning=styled.h4`
+text-align: center;
+  padding:0.8rem;
+  border-bottom:1px solid black;
+  margin:0;
+ 
+  `
+const Description=styled.p`
+  font-size:0.8rem;
```
Comment (sahil-shubham) [inline apps/http-status-codes/src/App.jsx:54]: Let's increase the font size of the description a bit. It is after all, the thing the user wanted to search for in the first place, what do you think?
```suggestion
  font-size:1.25rem;
```

**Review by sahil-shubham — APPROVED** [review]
@Reckon77 This works perfectly, amazing work. This would save us the trip to Mozilla's website.

File: apps/http-status-codes/src/files/codes.js (hunk 1 above)
Comment (Reckon77) [reply apps/http-status-codes/src/files/codes.js:7]: Ya, I have manually copied and pasted the information from the MDN website😅

File: apps/http-status-codes/src/App.jsx (hunk 2 above)
Comment (Reckon77) [reply apps/http-status-codes/src/App.jsx:54]: ya Sure

File: apps/http-status-codes/src/files/codes.js (hunk 1 above)
Comment (sahil-shubham) [reply apps/http-status-codes/src/files/codes.js:7]: Hard work! Thanks for putting in the time.

**Review by sahil-shubham — APPROVED** [review]
Merging this now, thanks for contributing another app @Reckon77

**Pull request merged**